blog/2015/03/instructor-debriefing-2015-03-10.html

```diff
@@ -0,0 +1,103 @@
+---
+layout: blog
+root: ../../..
+author: Sheldon McKay and Rayna Harris
+title: "2015 Post-workshop Instructor Debriefing, Round 4"
+date: 2015-03-13
+time: "10:00:00"
+category: ["Workshops"]
+---
+
+
+<!-- start excerpt -->
+<p>We had our fifth round of post-workshop debriefing this week.
+We discussed workshops help at the New York Academy of Sciences,
+the University of Oslo and ther University of British Columbia.  
+This was a very instructive meeting with important lessons learned
+from the perspective of both new and veteran instructors. One of the 
+key take-home lessons is that new instructors would benefit from attending
+an instructor debriefing prior to doing their first workshop.
+<!-- end excerpt -->
+
+Complete notes for this meeting can be found at https://etherpad.mozilla.org/KY6HBTIu57.
+
+<p>
+<h3>Major Lessons Learned from Debriefing Round 5</h3>
+<ul>
+<li>Good communication with the host and amongst instructors is critical for running a smooth workshop.  New instructors need
+clear guidance on what to expect and how to plan the lessons.</li>
+<li>We should remind beginning instructors that some time pressure is normal and it is not unusual to make it through less of the lesson than anticipated.</li>
+<li>There is some ambiguity as to what a SWC workshop should teach and what flexibility the members deciding what 
```

Comment:
I think it would be good to link to Greg's recent blog post on what to teach in workshops so that those interested can learn more.

http://software-carpentry.org/blog/2015/02/improving-instruction.html